### DISCUSSION TITLE
The commonName field is not allowed.

### DISCUSSION THREAD

**Discussion by 78Lefty** [Sleepw4lker/TameMyCerts#13] [Q&A] [answered]
Hello,

first thank you for your really good tool.

I installed a policy based on the computer sid template, now i get Request denied reason is The commonName field is not allowed.
I cant figure out why. Can anyone give me a hint?

**Comment by Sleepw4lker** [comment]
Hello @78Lefty this means your certificate request contains this field (commonName) and you must define a rule to allow this (everything which is not defined is automatically denied).

Have a look at the documentation and other examples:
https://github.com/Sleepw4lker/TameMyCerts/blob/main/user-guide/subject-rules.adoc
https://github.com/Sleepw4lker/TameMyCerts/blob/main/examples/Sample_Offline_Webserver.xml

Kind regards

**Comment by 78Lefty** [comment]
Hello,

Thanks for your very quick answer,  i added it, now i get
The commonName field is not allowed.
The mandatory commonName field was not found in the request.
... i did not get it.

**Comment by Sleepw4lker** [comment, marked as answer]
May it be that you placed the rule into the _SubjectAlternativeName_ section instead of the _Subject_ section of your config file? commonName is a field which occurs in the "Subject Distinguished Name". userPrincipalName (as used in the example) is a field that occurs in the "Subject Alternative Name" of a certificate or certificate request, so these two happen at different places.
